sound/soc/intel/boards/tgl_rt1308.c

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: GPL-2.0
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+
+/*
+ * tgl_rt1308.c - ASoc Machine driver for Intel platforms
+ * with RT1308 codec.
+ */
+
+#include <linux/acpi.h>
+#include <linux/delay.h>
+#include <linux/module.h>
+#include <linux/platform_device.h>
+
+#include <sound/core.h>
+#include <sound/jack.h>
+#include <sound/pcm.h>
+#include <sound/pcm_params.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+
+#include "../../codecs/rt1308.h"
+#include "../../codecs/hdac_hdmi.h"
+
+struct tgl_card_private {
+	struct list_head hdmi_pcm_list;
+};
```

Comment:
why do you declare this when there are no dailinks with HDMI?

Is this really intentional to only provide support for DMIC + amplifier?